spec/rails_helper.rb

```diff
@@ -1,13 +1,13 @@
 ENV['RAILS_ENV'] ||= 'test'
 require 'factory_girl_rails'
-require 'spec_helper'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
 require 'capybara/rails'
 require 'rack_session_access/capybara'
 
 ActiveRecord::Migration.maintain_test_schema!
 RSpec.configure do |config|
+  config.include Devise::TestHelpers, type: :controller
```

Comment:
Nice, didn't know you could limit inclusions like that. So does this mean we should do the opposite with the Warden test helpers?

```ruby
config.include Warden::Test::Helpers, type: :feature
```